netmiko/extreme/extreme_ssh.py

```diff
@@ -1,13 +1,123 @@
 """Extreme support."""
 from __future__ import unicode_literals
+import re
+import time
 from netmiko.cisco_base_connection import CiscoSSHConnection
 
 
+QUESTIONS = {
+    'save': 'Do you want to save configuration to',
+}
+
+
 class ExtremeSSH(CiscoSSHConnection):
     """Extreme support."""
+
     def session_preparation(self):
-        """Extreme requires enable mode to disable paging."""
+        """
+        Extreme requires enable mode to disable paging.
+        """
         self._test_channel_read()
-        self.enable()
         self.set_base_prompt()
         self.disable_paging(command="disable clipaging\n")
+
+    def answer_questions(self, *args, **kwargs):
+        """
+        Send questions/answers with send_command_timing.
+        """
+        if len(args) >= 1:
+            command_string = args[0]
+
+        # handle commands which return a question
+        output = self.send_command_timing(command_string)
+        if QUESTIONS[command_string] in output:
+            output += self.send_command_timing('yes')
+
+        return output
+
+    def send_command(self, *args, **kwargs):
+        """
+        Catch questions from the CLI.
+        Handle prompt changes during command executions.
+        """
+        if len(args) >= 1:
+            command_string = args[0]
+
+        # command that prompt a question
+        if command_string in QUESTIONS.keys():
+            output = self.answer_questions(command_string)
```

Comment:
I believe there are more cases that will require answers, at least on this platform, so I am not sure if only adding something for the 'save' will be enough. I've seen things like:
```
Warning: Entry exists. Are you sure you want to modify it? (y/N) Yes
```
That's why I initially thought about catching the `(y/N)` part, which could work on other platforms also (with some variation on the exact string).

Let me get back to you on this after some more testing, maybe I can come up with something more generic.